edx_ace/renderers.py

```diff
@@ -81,3 +81,21 @@ class EmailRenderer(AbstractRenderer):
     A renderer for :attr:`.ChannelType.EMAIL` channels.
     """
     rendered_message_cls = RenderedEmail
+
+
+@attr.s
+class RenderedPushNotification:
+    """
+    Encapsulates all values needed to send a :class:`.Message`
+    over an :attr:`.ChannelType.PUSH`.
+    """
+
+    subject = attr.ib()
```

Comment:
I think using `subject` would be confusing since it's an email term.

Using `title` is more appropriate and consistent with the mobile experience.

Sorry it'll require changing many variable and file names, but I hope it's useful.

```suggestion
    title = attr.ib()
```

Reply:
Changed